packages/blade/src/components/Checkbox/CheckboxIcon/Fade.native.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import type { EasingFn } from 'react-native-reanimated';
+import Animated, { Keyframe } from 'react-native-reanimated';
+import type { CSSObject } from 'styled-components';
+import styled from 'styled-components/native';
+import { useTheme } from '../../BladeProvider';
+
+type FadeProps = {
+  children: React.ReactNode;
+  show?: boolean;
+  styles: CSSObject;
+};
```

Comment:
this breaks the whole assumption of cross platform because the API itself has changed for web and native. on native TSC will yell since we are exposing the props of `web` and the props for web look something like this

```
type FadeProps = {
  show?: boolean;
  children: React.ReactNode;
  enter: string;
  leave: string;
  styles?: React.CSSProperties;
};
```

enter and leave are required for web.

Reply:
Will fix it.

Reply:
Fixed.